.gitignore

```diff
@@ -1,101 +1,4 @@
-# https://github.com/github/gitignore/blob/master/Python.gitignore
-.# Byte-compiled / optimized / DLL files
 __pycache__/
-*.py[cod]
-*$py.class
-
-# C extensions
-*.so
-
-# Distribution / packaging
-.Python
-build/
```

Comment:
I think we want to keep skipping the build dir?  At least I was using it before I think

Reply:
Looks like it's not being made anymore